src/test/resources/Features/OCD-1566-updateDefaultSearchParametersForRecentlyRetiredACBs.feature

```diff
@@ -0,0 +1,45 @@
+@Regression
+Feature: OCD-1566 - Update default search parameters for recently retired ACBs
+  This test checks whether CCHIT is unchecked on Search page and ACBs can be retired and unretired 
+
+  Scenario: Verify CCHIT checkbox is unchecked on Search page
+    Given I am on CHPL search page
+    And I look at ACB filter options selected for default search
+    Then I see that "CCHIT" checkbox is unchecked
+
+  Scenario Outline: Verify ACBs can be retired
+    Given I'm logged in as "ROLE_ADMIN"
+    When I navigate to ONC ACB Management page
+    And I click on "<ACB Name>"
+    And I edit an ACB 
+    And I mark it as retired on "08/01/2015"
+    Then ONC ACB have retired status as "Yes"
+    Examples:
+      |ACB Name      |
+      |InfoGard      |
+      |ICSA Labs     |
+      |Drummond Group|
+      |SLI Compliance|
+
+  Scenario Outline: Verify ACBs can be unretired
```

Comment:
Something is weird with these scenarios. You shouldn't be able to unretire ACBs that are currently not retired. If these scenarios assume the "retire an ACB" scenario has happened then that's not okay

Reply:
If you see the earlier scenario, I have retired the ACBs so in this scenario I'm just marking it as unretired. So that in the next nightly AQA execution, the test will again retire the ACBs

Reply:
One of the basic ideas of a test is that it should never depend on any other tests having happened or not. Tests should be able to be run in any order, skipping any or all of them, and all still work correctly. If one test depends on another test then that test is wrong